tests/functional/aws-node-sdk/test/bucket/aclUsingPredefinedGroups.js

```diff
@@ -0,0 +1,634 @@
+const assert = require('assert');
+const AWS = require('aws-sdk');
+const { errors } = require('arsenal');
+
+const withV4 = require('../support/withV4');
+const BucketUtility = require('../../lib/utility/bucket-util');
+const constants = require('../../../../../constants');
+const { VALIDATE_CREDENTIALS, SIGN } = AWS.EventListeners.Core;
+
+withV4(sigCfg => {
+    const ownerAccountBucketUtil = new BucketUtility('default', sigCfg);
+    const otherAccountBucketUtil = new BucketUtility('lisa', sigCfg);
+    const s3 = ownerAccountBucketUtil.s3;
+
+    const testBucket = 'predefined-groups-bucket';
+    const testKey = '0.txt';
+    const ownerObjKey = 'account.txt';
+    const testBody = '000';
+
+    function unauthenticatedRequest(operation, params, callback) {
+        const bucketUtil = new BucketUtility('default', sigCfg);
+        const request = bucketUtil.s3[operation](params);
+        request.removeListener('validate', VALIDATE_CREDENTIALS);
+        request.removeListener('sign', SIGN);
+        request.send(callback);
+    }
+
+    // tests for authenticated user(signed) and anonymous user(unsigned)
+    [true, false].forEach(auth => {
+        const authType = auth ? 'authenticated' : 'unauthenticated';
+        const grantUri = `uri=${auth ?
+            constants.allAuthedUsersId : constants.publicId}`;
+
+        describe('PUT Bucket ACL using predefined groups - ' +
+            `${authType} request`, () => {
+            beforeEach(done => s3.createBucket({
+                Bucket: testBucket,
+            }, err => {
+                assert.ifError(err);
+                return s3.putObject({
+                    Bucket: testBucket,
+                    Body: testBody,
+                    Key: ownerObjKey,
+                }, done);
+            }));
+            afterEach(() => ownerAccountBucketUtil.empty(testBucket)
+                .then(() => ownerAccountBucketUtil.deleteOne(testBucket)));
+
+            it('should grant read access', done => {
+                s3.putBucketAcl({
+                    Bucket: testBucket,
+                    GrantRead: grantUri,
+                }, err => {
+                    assert.ifError(err);
+                    const param = { Bucket: testBucket };
+                    const cb = er => {
+                        assert.ifError(er);
+                        done();
+                    };
```

Comment:
Instead of including this each time you could create a helper method

```js
function wrapCallback(done) {
  return err => {
    assert.ifError(err);
    done();
  };
}
```
Then use `wrapCallback(done)` in place of `cb` in all successful future cases.